dockerfiles/analytics/Dockerfile

```diff
@@ -30,12 +31,12 @@ ARG USER_HOME=/home/${USER}
 ARG JDK_NAME=jdk-8u251-linux-x64.tar.gz
 ARG JAVA_HOME=${USER_HOME}/java
 ARG JDK_URL
-ARG WSO2_SERVER=wso2ei
+ARG WSO2_SERVER_NAME=wso2ei
 ARG WSO2_SERVER_VERSION=6.2.0
 ARG WSO2_SERVER_REPOSITORY=product-ei
```

Comment:
@NishikaDeSilva use this in build arg `WSO2_SERVER_DIST_URL`.

Reply:
done